src/pages/sign-in/SignIn.tsx

```diff
@@ -51,7 +59,16 @@ const SignIn = () => {
     },
   });
 
-  const handleSubmitLogin = (body: LoginBody) => login(body);
+  const handleSubmitLogin = (body: LoginBody) => {
+    if (checkedRemember) {
+      localStorage.setItem(
+        "savedAccount",
```

Comment:
You should store **savedAccount** in constants file

Reply:
Only call when its needed